Tests/v3LoadShedderIntegration/v3LoadShedderIntegrationTests/v3LoadShedderIntegrationTests.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  v3LoadShedderIntegrationTests.swift
+//
+//  Created by Andrés Boedo on 3/29/23.
+
+@testable import Purchases
+import StoreKit
+import StoreKitTest
+@testable import v3LoadShedderIntegration
+import XCTest
+
+final class v3LoadShedderIntegrationTests: XCTestCase {
```

Comment:
Looks like tests don't like this name?

Reply:
forgot to push the line with the fix, it just doesn't like class names starting with lowercase